packages/telemed-ehr/app/src/telemed/features/appointment/MedicalHistoryTab/MedicalConditions/MedicalConditionsProviderColumn.tsx

```diff
@@ -75,9 +75,9 @@ export const MedicalConditionsProviderColumn: FC = () => {
         )}
       </Box>
 
-      {conditions.length === 0 && isReadOnly && !isChartDataLoading && (
+      {/* {conditions.length === 0 && isReadOnly && !isChartDataLoading && (
```

Comment:
Either should remove, or would be nice to have an explanatory comment, same for knownallergies and current medications.